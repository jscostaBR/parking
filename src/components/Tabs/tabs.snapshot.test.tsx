import React from 'react'
import { Tabs } from '.'
import renderer from 'react-test-renderer'

const tabList = [
  {
    tabName: 'TestComponent0',
    isActive: true,
    component: <div>TestComponent0</div>,
  },
  {
    tabName: 'TestComponent1',
    isActive: false,
    component: <div>TestComponent1</div>
  }
]

describe('render Tabs', () => {
  it('render Tabs with data', () => {
    const tree = renderer.create(<Tabs dependencies={{tabList}}/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
