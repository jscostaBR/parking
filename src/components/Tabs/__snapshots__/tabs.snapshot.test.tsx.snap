// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Tabs render Tabs with data 1`] = `
<div
  className="sc-bczRLJ kCaXxT"
>
  <nav
    className="sc-gsnTZi fcWxDb"
  >
    <button
      className="sc-dkzDqf cLImuM"
      onClick={[Function]}
    >
      TestComponent0
    </button>
    <button
      className="sc-dkzDqf jwdAEe"
      onClick={[Function]}
    >
      TestComponent1
    </button>
  </nav>
  <main
    className="sc-hKMtZM"
  >
    <div>
      TestComponent0
    </div>
  </main>
</div>
`;
